cmd/lifecycle/main.go

```diff
@@ -189,8 +190,8 @@ func verifyBuildpackApis(group buildpack.Group) error {
 	for _, bp := range group.Group {
 		if bp.API == "" {
 			// if this group was generated by this lifecycle bp.API should be set
-			// but if for some reason it isn't default to 0.2
-			bp.API = "0.2"
+			// but if for some reason it isn't default to latest
+			bp.API = api.Buildpack.Latest().String()
```

Comment:
I wonder if we should think about failing in this case instead of defaulting.

Reply:
Yeah... that's probably fair. The lifecycle has ensured there is an API version here for many years now!

Reply:
Updated in fb806d31